src/client/interpreter/locators/services/conda.ts

```diff
@@ -0,0 +1,6 @@
+import { IS_WINDOWS } from "../../../common/utils";
+
+// where to find the Python binary within a conda env
+export const CONDA_RELATIVE_PY_PATH = IS_WINDOWS ? ['python.exe'] : ['bin', 'python'];
+export const AnacondaCompanyName = 'Continuum Analytics, Inc.';
```

Comment:
This is actually no longer true as they changed their name to "Anaconda, Inc.": https://www.anaconda.com/blog/company-blog/anaconda-a-coming-of-age-story/.